lib/db/schemaupdater.go

```diff
@@ -107,10 +99,11 @@ func (db *schemaUpdater) updateSchema() error {
 		return err
 	}
 
-	return nil
+	l.Infoln("Compacting database after migration...")
+	return db.Compact()
 }
 
-func (db *schemaUpdater) updateSchema0to1() error {
+func (db *schemaUpdater) updateSchema0to1(_ int) error {
```

Comment:
Nit: Given the new argument the previous version in the name isn't really correct anymore.

Reply:
Mmm. Only added to support the weirdness in the to-v5 migrations... But I guess since I'm anyway touching the function signatures I might as well remove the "from" part in the name.